### ISSUE TITLE
SMA Wiki - Small correction to Git for Windows instruction

### ISSUE THREAD

**Issue by sm18lr88** [supermemo/SuperMemoAssistant.Documentation#77]
#### Replace <> with ""
SMA wiki → internet backups instructions → 5th step says "In the command prompt, type git clone < https://github.com/...... >"  I was only able to get this command to work by using quotation marks ("https://...")  instead of the < https://.... > 
Consider replicating and correcting the text.
Otherwise the rest of the instructions worked out.

#### Environment 
- **OS**: Windows 10 64bit
- **OS Language**: English

**Comment by alexis-** [comment]
I agree that this is confusing. The <> bit is meant as a placeholder
